databridge.examples/databridge.examples.dot-aas-v3-multiroute/src/test/java/org/eclipse/digitaltwin/basyx/databridge/dotaasv3multicast/test/TestAASUpdater.java

```diff
@@ -0,0 +1,172 @@
+import static org.mockserver.model.HttpResponse.response;
+
+import java.io.IOException;
+
+import org.apache.http.HttpStatus;
+import org.eclipse.digitaltwin.basyx.databridge.aas.configuration.factory.AASProducerDefaultConfigurationFactory;
+import org.eclipse.digitaltwin.basyx.databridge.core.component.DataBridgeComponent;
+import org.eclipse.digitaltwin.basyx.databridge.core.configuration.factory.RoutesConfigurationFactory;
+import org.eclipse.digitaltwin.basyx.databridge.core.configuration.route.core.RoutesConfiguration;
+import org.eclipse.digitaltwin.basyx.databridge.jsonata.configuration.factory.JsonataDefaultConfigurationFactory;
+import org.eclipse.digitaltwin.basyx.databridge.paho.configuration.factory.MqttDefaultConfigurationFactory;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.eclipse.paho.client.mqttv3.MqttPersistenceException;
+import org.eclipse.paho.client.mqttv3.MqttSecurityException;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.Header;
+import org.mockserver.verify.VerificationTimes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.moquette.broker.Server;
+import io.moquette.broker.config.ClasspathResourceLoader;
+import io.moquette.broker.config.IConfig;
+import io.moquette.broker.config.IResourceLoader;
+import io.moquette.broker.config.ResourceLoaderConfig;
+
+public class TestAASUpdater {
+	private static final String INPUT_DATA = "{\"Process\": {\"ProcessID\": \"00001\",\"Result\": true,\"ProcessData\": {\"Volume_NV\": 410.5,\"Volume_AV\": 407.7,\"Charge\": \"0000000001\",\"Pressure\": \"43.31\",\"Speed\": \"0.158790365\",\"Temperature\": \"42\",\"Result\": \"1\",\"StartProcess\": \"2022-06-22T08:21:33.4300238Z\",\"Volume\": \"39.7\",\"Weight\": \"49.69\",\"EndPorcess\": \"2022-06-22T08:22:56.4085953Z\"}}}";
+
+	private static final String SUBMDOEL_ONE_PATH = "/submodels/b25l/submodel-elements/duration/$value";
+	private static final String SUBMODEL_OTHER_PATH = "/submodels/b3RoZXI=/submodel-elements/duration/$value";
+	private static final String PROPERTY_PROCESS_DURATION_VALUE = "\"1M22S\"";
+
+	private static Logger logger = LoggerFactory.getLogger(TestAASUpdater.class);
+
+	private static DataBridgeComponent updater;
+	protected static Server mqttBroker;
+
+	private static ClientAndServer mockServer;
+
+	@BeforeClass
+	public static void setUp() throws IOException {
+		configureAndStartMockserver();
+
+		configureAndStartMqttBroker();
+
+		configureAndStartUpdaterComponent();
+		try {
+			publishNewDatapoint("process-update");
+			waitForPropagation();
+		} catch (MqttException | InterruptedException e) {
```

Comment:
Yor're right, would not hurt. But since this is just a test and the exception is not expected, we can ignore it imo.

Reply:
Not sure if I got this wrong but I thought doing it like this would mean that when the catch is triggered the program would still continue even if it shouldn't.